securedrop/tests/functional/functional_test.py

```diff
@@ -73,7 +85,7 @@ def _create_firefox_webdriver(self, log_file):
 
     def _create_webdriver(self):
         log_file_path = 'tests/log/firefox.log'
-        abs_log_file_path = os.path.abspath(log_file_path)
+        abs_log_file_path = os.path.abspath(os.path.join(__file__, '../../log/firefox.log'))
```

Comment:
My bad earlier, but this should be `abspath(join(dirname(__file__), '../log/firefox.log'))`. Have two `..` works, but is a little funky logically because then we're calling `abspath` on `/.../securedrop/tests/functional/functional_test.py/../../log/firefox.log`.